src/common/src/telemetry/mod.rs

```diff
@@ -39,8 +39,13 @@ pub const TELEMETRY_CLUSTER_TYPE_KUBERNETES: &str = "kubernetes";
 pub const TELEMETRY_CLUSTER_TYPE_SINGLE_NODE: &str = "single-node";
 pub const TELEMETRY_CLUSTER_TYPE_DOCKER_COMPOSE: &str = "docker-compose";
 const TELEMETRY_CLUSTER_TYPE_TEST: &str = "test";
+pub use risingwave_telemetry_event::{
+    TELEMETRY_RISINGWAVE_CLOUD_UUID, get_telemetry_risingwave_cloud_uuid,
+};
 
-pub use risingwave_telemetry_event::get_telemetry_risingwave_cloud_uuid;
+pub fn is_cloud_hosted() -> bool {
+    get_telemetry_risingwave_cloud_uuid().is_some()
```

Comment:
Wondering which is the most appropriate way to decide if we are running on Cloud. FYI, we also use `RISINGWAVE_CLOUD` here: https://github.com/risingwavelabs/risingwave/blob/4a1100e359469e401e044a542f64d40911938844/src/common/src/util/deployment.rs#L31-L42

cc @arkbriar

Reply:
I believe `RISINGWAVE_CLOUD` is the right way.